src/lib/ssg.tsx

```diff
@@ -1,122 +1,94 @@
-import fs from "fs";
-import { sync } from "glob";
-import { join, relative } from "path";
-import matter from "gray-matter";
-
 import { createClient, Entry } from "contentful";
-import { Document } from '@contentful/rich-text-types';
+import { Document } from "@contentful/rich-text-types";
 
 export interface DocMeta {
   title: string;
   chapter: string;
   subchapter: string;
   section: string;
   content?: string;
-  slug?: string | string[];
-  post?: Document
+  slug?: string;
+  post?: Document;
 }
 
 interface slugRetrieved {
-  slug: string
+  slug: string;
 }
 
-// Queries the list of slugs
-export const getDocSlugs = (subDir: string): string[] => {
-  const docsDirectory = join(process.cwd(), `docs/${subDir}`);
-  const files = sync(join(docsDirectory, "**/*.md"));
-  const fileNames = files.map((file) => relative(docsDirectory, file));
-  return fileNames;
-};
-
 export async function getDocSlugsFromCMS(): Promise<string[]> {
   const client = createClient({
     space: process.env.NEXT_PUBLIC_CONTENTFUL_SPACE_ID,
     accessToken: process.env.NEXT_PUBLIC_CONTENTFUL_ACCESS_KEY,
   });
 
   const res = await client.getEntries<slugRetrieved>({
-    content_type: "admissions", 
-    select: "fields.slug"
-  })
+    content_type: "admissions",
+    select: "fields.slug",
+  });
 
-  return res.items.map((x: Entry<slugRetrieved>) => x.fields.slug);
-};
+  return res.items.map((x: Entry<slugRetrieved>) => {
+    return x.fields.slug;
+  });
+}
 
-export async function getDocBySlugFromCMS(slug: string | string[]): Promise<DocMeta> {
+export async function getDocsBySlugsFromCMS(
+  slug: string | string[]
+): Promise<DocMeta[]> {
   const client = createClient({
     space: process.env.NEXT_PUBLIC_CONTENTFUL_SPACE_ID,
     accessToken: process.env.NEXT_PUBLIC_CONTENTFUL_ACCESS_KEY,
   });
 
   if (Array.isArray(slug)) {
-    slug = slug.join('-');
+    slug = slug.join(",");
```

Comment:
Ini kenapa y

Reply:
soalnya kalo pake `-`, susah diliat firstly since before-01-before-02, secondly, harusny [in] bukan [match] (soalnya yang lu pass kesana itu udah array of slugs kayak before-01, before-02, etc, bukan [before, 01])

Reply:
oh okk